app/src/main/res/values/strings.xml

```diff
@@ -339,12 +339,12 @@
     <string name="other">Other</string>
     <string name="compression">Compression</string>
     <string name="compression_none">None</string>
-    <string name="compression_gzip">GZip</string>
-    <string name="recycle_bin">Recycle Bin</string>
+    <string name="compression_gzip">.gzip</string>
```

Comment:
gzip

Reply:
"gzip" (what is in the string currently) is what it is called. ".gzip" avoids the problem of it being either "Gzip" or "gzip" ? Why it's a problem?